mods/persistence/modules/world_save/serializers/_serializer.dm

```diff
@@ -56,6 +56,9 @@
 	// We check to see if this is a reference only var so that if things are missing from the resolver, this doesn't fail silently.
 	if(reference_only && !resolver.things["[object_id]"])
 		return null
+	if(!istype(resolver.things["[object_id]"], /datum/persistence/load_cache/thing))
```

Comment:
When, if ever, would this occur?

Reply:
Every times I'd load from save actually? 
Kept getting things trying to access non-existent vars there. And I couldn't explain it. Its like there's shared ids.